modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -5,13 +5,15 @@
 from __future__ import absolute_import
 
 import unittest
+import datetime
+from io import StringIO
```

Comment:
What if we delete this import and have bytes examples use this?
var = bytes(b'blah')
Note: this will work for python 2.7 and later

Reply:
Done